test/functional/assays_controller_test.rb

```diff
@@ -1860,4 +1860,62 @@ def check_fixtures_for_authorization_of_sops_and_datafiles_links
     end
   end
 
+  test 'last updated by - content' do
+    person1 = Factory(:person)
+    person2 = Factory(:person)
+    a = Factory :assay, policy: Factory(:public_policy), created_at: 15.minute.ago, contributor: person1
+    Factory :activity_log, activity_loggable: a, action: 'create', created_at: 15.minute.ago, culprit: person1
+    login_as(person1)
+    get :show, params: { id: a.id }
+    assert_response :success
+    assert_select 'span.updated_last_by', false, 'Last editor should not be shown just after creation'
+    Factory :activity_log, activity_loggable: a, action: 'update', created_at: 10.minute.ago, culprit: person1
+    get :show, params: { id: a.id }
+    assert_response :success
+    assert_select 'span.updated_last_by', person1.name
```

Comment:
this would be better if the css selector was `span.updated_last_by a`, and the check was `text:person1.name`

you should also check the link, with `span.updated_last_by a[href=?]` and check for `person_path(person1)`